environment/src/main/resources/schema/app/20210817101522_CB-13650_enablemultiaz_flag_added_to_freeiparequest.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-13650 enablemultiaz flag added to freeiparequest
```

Comment:
Nit: perhaps update the comment according to the PR / commit title `Integrate Multi AZ with FreeIPA controller layer`.